modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+
+.. Create an object YAML file to define the local volume set, such as `local-volume-set.yaml`, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: local.storage.openshift.io/v1alpha1
+kind: LocalVolumeSet
+metadata:
+  name: example-autodetect
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+          - key: kubernetes.io/hostname
+            operator: In
+            values:
+              - worker-0
+              - worker-1
+  storageClassName: example-storageclass <1>
+  volumeMode: Filesystem
+  fstype: ext4
+  maxDeviceCount: 10
+  deviceInclusionSpec:
+    deviceTypes: <2>
+      - disk
+      - part
+    deviceMechanicalProperty:
+      - NonRotational
+    minSize: 10G
+    maxSize: 100G
+    models:
+      - SAMSUNG
+      - Crucial_CT525MX3
+    vendors:
+      - ATA
+      - ST2000LM
+status:
+  conditions:
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: 'DiskMaker: Available, LocalProvisioner: Available'
+    status: "True"
+    type: DaemonSetsAvailable
+  - lastTransitionTime: "2020-10-12T16:48:47Z"
+    message: Operator reconciled successfully.
+    status: "True"
+    type: Available
+  observedGeneration: 1
+  totalProvisionedDeviceCount: 0
+----
++
+<1> Determines the storage class that is created for persistent volumes that are provisioned from discovered devices.
++
+<2> When using the local volume set feature, the Local Storage Operator does not support the use of logical volume management (LVM) devices.
+
+.. Create the local volume set object:
++
+[source,terminal]
+----
+$ oc apply -f local-volume-set.yaml
+----
+
+.. Verify that the local persistent volumes were dynamically provisioned based on the storage class:
++
+[source,terminal]
+----
+$ oc -n get pv
+----
++
+.Example output
+[source,terminal]
+----
+NAME                CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM   STORAGECLASS           REASON   AGE
+local-pv-1cec77cf   100Gi      RWO            Delete           Available           example-storageclass            88m
+local-pv-2ef7cd2a   100Gi      RWO            Delete           Available           example-storageclass            82m
+local-pv-3fa1c73    100Gi      RWO            Delete           Available           example-storageclass            48m
+----
+
+[NOTE]
+====
+Results are deleted after they are removed from the node. Symlinks must be manually removed.
+====
+
+[discrete]
+== Hiding the Local Volume Discovery Result tab
+
+The Local Volume Discovery Result tab is an API output list used for local volume discovery. It is not necessary for {product-title} users to have access to this tab.
+
+As a cluster administrator, you can hide the tab by following the instructions in "Hiding internal objects". See xref:../../operators/operator_sdk/osdk-generating-csvs.adoc#osdk-generating-csvs[Generating a ClusterServiceVersion (CSV)] for more information.
```

Comment:
@gnufied I added this to the end of the section. WDYT?

Reply:
We don't need this in docs, this is part of internal detail of the BZ!